docs/sources/next/javascript-api/k6-experimental/browser/locator/clear.md

```diff
@@ -0,0 +1,52 @@
+excerpt: 'Browser module: locator.clear method'
+---
+
+# clear([options])
+
+Clears text boxes and input fields (`input`, `textarea` or `contenteditable` elements) of any existing values.
+
+<TableWithNestedRows>
+
+| Parameter           | Type    | Default | Description                                                                                                                                                                                                                                                                                                                                   |
+| ------------------- | ------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| options             | object  | `null`  |                                                                                                                                                                                                                                                                                                                                               |
+| options.force       | boolean | `false` | Setting this to `true` will bypass the actionability checks (`visible`, `stable`, `enabled`).                                                                                                                                                                                                                                                 |
+| options.noWaitAfter | boolean | `false` | If set to `true` and a navigation occurs from performing this action, it will not wait for it to complete.                                                                                                                                                                                                                                    |
+| options.timeout     | number  | `30000` | Maximum time in milliseconds. Pass `0` to disable the timeout. Default is overridden by the `setDefaultTimeout` option on [BrowserContext](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/browsercontext/) or [Page](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/page/). |
+
+</TableWithNestedRows>
+
+### Example
+
+{{< code >}}
+
+```javascript
+import { browser } from 'k6/experimental/browser';
+
+export const options = {
+  scenarios: {
+    browser: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+};
+
+export default async function () {
+  const page = browser.newPage();
+
+  await page.goto('https://test.k6.io/browser.php');
+
+  page.locator('#text1').clear();
```

Comment:
Showing an input field filled out and printing its value might provide a better example for users. Then, we could clear the field and reprint its value to allow users to see it cleared.

Reply:
How about the [example](https://github.com/grafana/k6/blob/release-v0.49.0/release%20notes/v0.49.0.md#add-clear-to-the-locator-class-browser1149) from the upcoming release notes?

Reply:
Looks good 👍

Reply:
Updated in 9b431919079ed428cb15e0be259be3a68e93d31b